bit_manipulation/Count_of_set_bits.cpp

```diff
@@ -0,0 +1,31 @@
+/** * @file
+ * @brief program to Count set bits in an integer 
+ * @details
+ * Time Complexity :- O(log n)
+ * Space complexity :- O(1)
+
+**/
+#include <iostream>
+using namespace std;
+ 
+/* Function to get no of set bits in binary
+representation of positive integer n */
+unsigned int countSetBits(unsigned int n)
+{
+    unsigned int count = 0; // "count" variable is used to count number of 1's in binary representation of the number

```

Comment:
Consider using uint64_t for non-negative values (or their appropriate size: uint32_t, uint16_t, uint8_t) or int64_t for negative values. Check other parts of the code ([reference](https://en.cppreference.com/w/cpp/types/integer)). 🙂

Reply:
I will surely do sir

Reply:
And easy way to achieve repository standards is to open this in gitpod, run `clang-tidy` & `clang-format` as given [here](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md). Further you can enable github-action to your fork for automating `clang-tidy` & `clang-format` after every push!